### DISCUSSION TITLE
Adding addresses to a host

### DISCUSSION THREAD

**Discussion by 9cb14c1ec0** [Performave/convoy#103] [Support]
Maybe I'm missing something, but I can' figure out how to set up addresses.  Can someone point me in the right direction?

**Comment by ericwang401** [comment]
Can you elaborate more on what result you are trying to achieve?

**Comment by 9cb14c1ec0** [comment]
I have a node set up in convoy.  There are no addresses assigned to it, and no obvious way to do that.  I want addresses assigned to the node that can be used for new VMs.  I have a /27 set up on promox.  How do I inform convoy about that /27?

**Reply by chrisvorer6** [reply]
Hi, have you solved this problem? I have the same problem now, can you help me?

> I have a node set up in convoy. There are no addresses assigned to it, and no obvious way to do that. I want addresses assigned to the node that can be used for new VMs. I have a /27 set up on promox. How do I inform convoy about that /27?

**Reply by 9cb14c1ec0** [reply]
Never got an answer, never was able to figure it out, so I've moved on from convoy.

**Comment by chrisvorer6** [comment]
I found it.
Convoy admin control panel
IPAM--Create Pool
I set name as test
Then click test, now you can see the Greate Address button.

9cb14c1ec0 ***@***.***> 于2024年11月30日周六 20:10写道：
